create_aio_app/template/{{cookiecutter.project_name}}/{{cookiecutter.project_name}}/utils/wait_script.py

```diff
@@ -0,0 +1,17 @@
+{%- if cookiecutter.use_postgres == 'y' %}
```

Comment:
U don't need `wait_script.py` without postgres (or redis in future), use hook for removing this file in final build https://github.com/aio-libs/create-aio-app/blob/master/create_aio_app/template/hooks/post_gen_project.py